Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM python:3.6-alpine
+
+#
+# Handle the env info
+#
+ARG BUILD_COMMIT_SHA1
+ARG BUILD_COMMIT_DATE
+ARG BUILD_BRANCH
+ARG BUILD_DATE
+ARG BUILD_REPO_ORIGIN
+ 
+ENV BUILD_COMMIT_SHA1=$BUILD_COMMIT_SHA1
+ENV BUILD_COMMIT_DATE=$BUILD_COMMIT_DATE
+ENV BUILD_BRANCH=$BUILD_BRANCH
+ENV BUILD_DATE=$BUILD_DATE
+ENV BUILD_REPO_ORIGIN=$BUILD_REPO_ORIGIN
+
+WORKDIR /src/
+ADD . /src/
+
+RUN apk update \
+    && apk add build-base \
+    jpeg-dev \
+    zlib-dev
+ENV LIBRARY_PATH=/lib:/usr/lib
+
+ENV PYTHONUNBUFFERED 1
+
+RUN pip install pip
+RUN python setup.py install
+
+# uWSGI will listen on this port
+EXPOSE 8000
+
+# # Call collectstatic (customize the following line with the minimal environment variables needed for manage.py to run):
+# RUN DATABASE_URL=none /venv/bin/python manage.py collectstatic --noinput
+
+# Start uWSGI
+ENTRYPOINT ["coderedcms"]
```

Comment:
Question... running the `coderedcms` command will not actually do anything by default. It is used as `coderedcms start` to create a new boilerplate project. So is this the most appropriate entrypoint command? I'm not really sure what the purpose of this docker image is.

Reply:
yes this is true. I suspect i may have miudnerstood the purpose of this repo. but regardless an interesting project.

Reply:
also using `coderedcms` as the entrypoint allows you to do things like `docker run --rm -v $PWD/clients/myclient:/src/output coderedcms:latest start`

Reply:
ps yes it is the valid entrypoint; allows you to run start in your docker compose or kube config

Reply:
`docker run --rm -v $PWD/clients/myclient:/src/output coderedcms:latest start my_project` which will output the generated to `/src/output`